Enhanced SlotRenderer GetSlotArgs (#7794)
plone/volto: docs/source/configuration/slots.md

<SlotRenderer name="aboveContent" content={content} />
```

These are the props that you can pass to the `SlotRenderer` component.

```ts
interface SlotRendererProps {
  name: string;
  content: Content;
  navRoot?: Content;
  data?: BlocksFormData;
  [key: string]: unknown;
}
```

> They are passed by to evaluate the predicates of the slot components and for passing it down to the rendered slot component.

Reviewer: This is a little bit confusing. Is this right?

```suggestion
The props are passed both to evaluate the predicates of the slot components and for passing them down to the rendered slot component.
```
